.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,11 @@
  
 ### Issues Resolved
 [List any issues this PR will resolve]
- 
+
+Functional test repo location:
```

Comment:
Could we support this by commenting in the PR

Reply:
If there are multiple reviews on the PR, those will all count as comments and trigger another run for no reason.